fury/tests/test_actors.py

```diff
@@ -537,6 +537,26 @@ def test_odf_slicer(interactive=False):
     del odf_actor
     del odfs
 
+def test_peak_symmetric(interactive=False):
+    _peak_dirs = np.array([[1, 0, 0], [0, 1, 0], [0, 0, 1]], dtype='f4')
+    peak_dirs = np.zeros((4, 4, 4, 3, 3))
+    peak_dirs[:, :, :] = _peak_dirs
+    peak_values = np.random.rand(4, 4, 4, 3)
+
+    scene = window.Scene()
+    peak_actor = actor.peak(peak_dirs, peaks_values=peak_values,symmetric=False)
+    scene.add(peak_actor)
+
+    if interactive:
+        window.show(scene)
+
+    report = window.analyze_scene(scene)
+    ex = ['vtkOpenGLActor']
+    npt.assert_equal(report.actors_classnames, ex)
```

Comment:
strange test, it would be great to test the symmetry. you have `symmetry=False` so you are not testing your new feature.

I am not sure what you are testing. Can you clarify?